src/main/java/io/seqera/tower/cli/responses/ComputeEnvs/ComputeEnvsPrimaryGet.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2021, Seqera Labs.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * This Source Code Form is "Incompatible With Secondary Licenses", as
+ * defined by the Mozilla Public License, v. 2.0.
+ */
+
+package io.seqera.tower.cli.responses.ComputeEnvs;
+
+import io.seqera.tower.cli.responses.Response;
+import io.seqera.tower.model.ComputeEnv;
+
+public class ComputeEnvsPrimaryGet extends Response {
+
+    public final String workspaceRef;
+    public final ComputeEnv computeEnv;
+
+    public ComputeEnvsPrimaryGet(String workspaceRef, ComputeEnv computeEnv) {
```

Comment:
When using the option `--json` the whole `computeEnv` object is show. I think that it would be better to here only pass the `computeEnv.id` and `computeEnv.name` so the `json` output is less verbose.

Reply:
Not sure about this, after all, you are requesting to VIEW a specific compute environment, just like we do with the VIEW command when requesting by ID. If a user wishes to just check what the primary CE is, she can just use the LIST command with the new star feature.

I believe this command is far beyond checking what my CE is. Let me know what you think.

Reply:
Also you can use a tool like jq to parse and eventually trim the results down

Reply:
But, do we want to trim the result? Maybe I wish to VIEW the full primary  CE object to pipe it to another command.

Reply:
No absolutely. For me the result of the cli should be complete: than it's up to the user to eventually trim the result down by piping the result into another command like `jq`. I am curious though about the difference between the export and the view command now. I'll do some tests